### DISCUSSION TITLE
Raspberry Pi 5 stability

### DISCUSSION THREAD

**Discussion by craigusus** [Nachtzuster/BirdNET-Pi#87] [General]
Hi all,

Is anyone else using a Pi 5? I have an 8GB one and I'm having horrible stability issues with it. I have to pull the power to it nearly every other day as I lose web and SSH connection. When I open the enclosure I have a red status LED rather than a green one and it's not doing anything/it appears powered off as the fan is not spinning. It's not network/wifi connectivity as I have tested with ethernet and wifi.
It wouldn't be too much of an issue (well it is!) but I have to go outside and trip a breaker in my solar control box to reset the power to the Pi.
I was wondering if anyone else has stability issues, even on other models of Pi.

**Comment by alexbelgium** [comment]
Hi ! I don't have a rpi5 but encountered similar issues in the pase with rpi3b+ and rpi4. Have you a good quality power supply? In the end I use a way oversized power supply and it made my system MUCH more stable

And a high quality sdxc card, made to accept frequent read/writes. My best one tested so far is "Samsung endurance pro"

Edit : seeing you mention solar box, how is it set? I am not an expert on those system but I suppose you have a buffer battery that provides power in a stable manner, fluctuations in power could perhaps be a cause

**Reply by Frono** [reply]
I have/do run Birdnet with Samsung Pro Endurance SDXC on Pi 4 and Pi 5 without issue, even when it is very hot outside (95 f). However, I do use the experimental Ram Drive feature. I have successfully had one instance run on a Sandisk Extreme SDXC for a couple of years before it failed.

I use a CanaKit AC power supply for all my instances. (45w).

My instances generally run without issue for months at a time, with the biggest reason for failure being power outage...

**Comment by craigusus** [comment]
Hi Alex/Lloyd,

Thanks for the replies. I think it could be a power issue as when I reboot, it seems to run fine for a day or 2. If it was the card, surely it would be apparent from the start?

I have a stable 12V from my solar control panel (I have 2x 12V 100Ah LiFePO4 batteries) so voltage fluctuation shouldn't be the issue. However... the buck converter is only 3A (or 2A, depending on what text you read on Amazon!). And just now read, the official [Raspberry Pi 27W USB-C Power Supply](https://thepihut.com/products/raspberry-pi-27w-usb-c-power-supply) has a `Custom mode for Raspberry Pi 5: 5V/5A`. So a new 12V to 5V converter may be needed. I'm using my solar power for this project and using the 12V direct rather than going through the 12V to 240V invertor.

The microSD card I'm currently running is a [64GB Lexar High-Performance 633x Class 10](https://www.amazon.co.uk/gp/product/B012PLTLVK). Apparently, the 633x means only up to 95MB/s write speeds. I may have to 'treat' myself to a more endurance-rated microSD card as opposed to faster read/write times. The Samsung Endurance Pro is only 100/40 read/write.

Thanks for the advice guys.

**Reply by craigusus** [reply]
Phew, busy few weeks at work! BUT, I have had a stable Pi 5 running.
I got another [12V to 5V converter](https://www.amazon.co.uk/dp/B09PTPFYC7) allowing up to 15A! I have the 5V output connected directly to the GPIO pins and have had no connectivity or performance issues. So I can now fix it in the new enclosure as a more permanent installation 🤞🏻

**Reply by dalogue1** [reply]
My RPi 5 4GB with MP600 NVMe and 27W power supply is rock solid so far.  I agree that it must have been the PSU.

**Comment by W9MDO** [comment]
I had been running into similar issues, my RPi5 locking up every few days randomly. I was using a high power multiple output charger (one of the new breed of 140 or 280 watt versions with a total of 6 or 8 A and C outputs. 
There's no way the RPi5 is even getting close to that power consumption, but I've begun to wonder if a "charger" may not have the stability needed, ie it may suffer from short term dropouts due to the switch mode nature of the supply. Of course this would not be a problem if you're using the charger to charge a battery based notebook or phone, but could certainly cause the sort of problems I'm seeing, 
I've bought the new genuine 45W Pi supply and will give that a shot to test my hypothesis.

**Comment by dalogue1** [comment]
My RPi5 is rock solid with the stock Raspberry Pi USB-C power supply.

**Comment by tdelattre** [comment]
Craig, would you care to share your solar installation configuration? It could help others to know what works and what doesn't. Thanks!

**Reply by craigusus** [reply]
I can give it a go...

I currently have a [Renogy 50W 12V Monocrystalline Panel](https://uk.renogy.com/50-watt-12-volt-monocrystalline-small-solar-panels/) on the roof of my garden shed, which is south-facing (in the UK).
That is connected to a [Renogy Wanderer 30A PWM Charge Controller](https://uk.renogy.com/renogy-wanderer-li-30a-pwm-charge-controller/).
Which is charging 2x [ECO-WORTHY 12V 20Ah LiFePO4 batteries](https://uk.eco-worthy.com/collections/12v-lifepo4-lithium-battery/products/lifepo4-12v-30ah-lithium-iron-phosphate-battery) in series (can't seem to find the 20Ah ones anymore).
I then have the 12V fed from the batteries to a [12/24V DC to 5V DC step-down converter,](https://www.amazon.co.uk/dp/B09PTPFYC7) which I then feed to the Pi 5 via a [USB-C cable](https://www.amazon.co.uk/dp/B09WQDKN3C).
The Pi connects to my network via WiFi on a [TP-Link External Access Point ](https://www.tp-link.com/uk/business-networking/outdoor-ap/eap110-outdoor/).

![IMG20250605153140](https://github.com/user-attachments/assets/e14f0b59-3e55-4fbd-a5b5-0fd71193f480)
![IMG20250605153125](https://github.com/user-attachments/assets/666b989e-df71-4770-be2a-e3a52b1c57f2)
